package.json

```diff
@@ -38,6 +38,7 @@
     "gulp-rename": "^1.2.2",
     "gulp-strip-code": "^0.1.4",
     "gulp-uglify": "^2.0.1",
```

Comment:
You can probably remove `gulp-uglify` since you replaced it with `gulp-uglify-es`